spec/rack/test_spec.rb

```diff
@@ -341,7 +341,7 @@ def close
 
   describe '#basic_authorize' do
     it 'sets the HTTP_AUTHORIZATION header' do
-      authorize 'bryan', 'secret'
+      basic_authorize 'bryan', 'secret'
```

Comment:
This modification is not related to this fix directly, right?

Reply:
Yes, exactly. I just noticed while I was working on the new tests.

Reply:
As `basic_authorize` is the alias of `authorize` and it is not harmful without changing, I feel we can create another PR.